Docker/start-testnet.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+function call_hook {
+    [[ -x hooks.sh ]] && ./hooks.sh "$@"
+}
+
+script_path=$(dirname $(readlink -f $0))
+[[ -e env.sh ]] &&
+    echo "=== Load environments from env.sh" &&
+    . ./env.sh
+
+if [[ ! -f config.ini ]]; then
+    echo "Missing config.ini file for nodeos in current directory"
+    exit 1
+fi
+
+set ${REPOSITORY:=cyberway}
+set ${BUILDTYPE:=latest}
+
+[[ -x shutdown.sh ]] && (
+    echo "=== Shutdown previous instance"
+    ./shutdown.sh; rm shutdown.sh)
+
+echo "=== Prepare scripts for manage testnet"
+[[ -f shutdown.sh ]] && rm shutdown.sh
+[[ -f down.sh ]] && rm down.sh
+[[ -f up.sh ]] && rm up.sh
+
+cat >shutdown.sh <<DONE
+#!/bin/bash
+startup/run-with-events.sh cleanup
+exit 0
+DONE
+
+cat >down.sh <<DONE
+#!/bin/bash
+startup/run-with-events.sh down
+exit 0
+DONE
+
+cat >up.sh <<DONE
+#!/bin/bash 
+startup/run-with-events.sh up
+exit 0
+DONE
+chmod a+x shutdown.sh down.sh up.sh
+
+CYBERWAY_IMAGE=$REPOSITORY/cyberway:$BUILDTYPE
+echo "=== Extract cyberway startup scripts from $CYBERWAY_IMAGE"
+rm -fR startup || true
+docker create --name extract-cyberway $CYBERWAY_IMAGE
+docker cp extract-cyberway:/opt/cyberway/startup startup
+docker rm extract-cyberway
+
+GOLOS_IMAGE=$REPOSITORY/golos.contracts:$BUILDTYPE
+echo "=== Extract Golos convertion scripts from $GOLOS_IMAGE"
+rm -fR scripts || true
+docker create --name extract-golos-scripts $GOLOS_IMAGE
+docker cp extract-golos-scripts:/opt/golos.contracts/scripts scripts
+docker rm extract-golos-scripts
+
+echo "=== Start cyberway containers"
+startup/run-with-events.sh
+sleep 10
+call_hook "nodeos-started"
+
+echo "=== Deploy cyberway & golos contracts"
+sudo docker run --rm --network cyberway_cyberway-net -ti $GOLOS_IMAGE /opt/golos.contracts/scripts/boot-sequence.py
+call_hook "contracts-loaded"
+
+if [[ -n "$GOLOS_DB" ]]; then
+    [[ -z "$CYBERWAY_DB" ]] && export CYBERWAY_DB=$(awk -F'[=#]' '/^chaindb_address\s*=/{sub(/^ *| *$/, "", $2); print $2;}' <${PWD}/config.ini)
+    echo "=== Convert data from Golos database $GOLOS_DB into $CYBERWAY_DB"
+    call_hook "convert-prepare"
+    time scripts/convert.py
+    if [[ -z "$SKIP_CONVERT_PUBLISH" ]]; then
+       time scripts/convert_publish.py
+       time scripts/update_childcount.py
+    fi
+    call_hook "convert-finish"
+    docker stop nodeosd
+    docker stop cyberway-notifier
+    docker start cyberway-notifier nodeosd
```

Comment:
Is it a good idea to manually stop/start containers here without docker-compose?

Reply:
As I see, the docker correctly starts the containers in this case. But you are right, and this needs to be clarified. I will do this as part of the next task.